requirements/bench.txt

```diff
@@ -24,4 +24,6 @@ tensorflow-estimator==2.3.0
 tensorflow>=2.3.1
 image-quality==1.2.6
 keras==2.2.4
-tensorflow-estimator==2.3.0
\ No newline at end of file
+tensorflow-estimator==2.3.0
+pystan==2.19.1.1
+prophet
```

Comment:
how come this is not installed when installing mixsimulator?

Reply:
I think we missed something but we work on it. 
Adding prophet in bench.txt is just for testing

Reply:
fbprophet need pystan, so how to make sure pystan is installed before prophet? 
otherwis, the problem may be about this :
WARNING:pystan:MSVC compiler is not supported

Reply:
From researches I made, pystan needs a C++ compiler other than MSVC as it is not supported.

Reply:
is that on windows? you can probably disable windows entirely for this experiment

Reply:
Yes but it needs a specific compiler for his PyStan requirement